examples/scripts/ppo-sentiment-t5-small.py

```diff
@@ -34,8 +33,7 @@
 #   - fp16 (mixed-precision) or fp32 (normal precision)
 #
 # To run it in each of these various modes, first initialize the accelerate
-# configuration with `accelerate config` then run the script with 
-# `accelerate launch ppo-sentiment-t5-small.py`
+# configuration with `accelerate config`
```

Comment:
also added this piece of instruction

Reply:
Added it again in [d0875b5](https://github.com/lvwerra/trl/pull/93/commits/d0875b5eab233993493d6390be2d2bfe5d2e707a)